examples/layers/examples/nonce_management_filler.rs

```diff
@@ -35,22 +35,20 @@ async fn main() -> Result<()> {
     // Create a provider with the signer.
     let rpc_url = anvil.endpoint().parse()?;
     let provider = ProviderBuilder::new()
-        // Add the `NonceManagerLayer` to the provider.
-        // It is generally recommended to use the `.with_recommended_layers()` method, which
-        // includes the `NonceManagerLayer`.
-        .layer(NonceManagerLayer)
+        // Add the `NonceFiller` to the provider.
+        .with_nonce_management()
         .signer(EthereumSigner::from(signer))
-        .on_reqwest_http(rpc_url)?;
+        .on_http(rpc_url)?;
 
     // Create an EIP-1559 type transaction.
     let tx = TransactionRequest::default()
         .with_from(alice)
         .with_to(vitalik.into())
         .with_value(U256::from(100))
-        // Notice that without the `GasEstimatorLayer`, you need to set the gas related fields.
-        .with_gas_limit(U256::from(21000))
-        .with_max_fee_per_gas(U256::from(20e9))
-        .with_max_priority_fee_per_gas(U256::from(1e9))
+        // Notice that without the `GasFiller`, you need to set the gas related fields.
+        .with_gas_limit(21_000)
+        .with_max_fee_per_gas(20_000_000_000)
+        .with_max_priority_fee_per_gas(1_000_000_000)
```

Comment:
@prestwich because of the change to `u128` using the syntax `20e9` is recognized as a float, what is the intended way of defining numbers for these fields?

Reply:
usually either grouped 0s or by mantissa and significand

20_000_000_000
2 * 10.pow(10)